app/AuthenticatedModule/Presenters/AdminPresenter.php

```diff
@@ -56,6 +62,14 @@ public function handleSaveDelegates() : void
         $this->redirect('this');
     }
 
+    public function handleSaveCandidates() : void
+    {
+        if ($this->queryBus->handle(new CandidatesCountQuery()) === 0) {
```

Comment:
Nemůže tohle být přímo v repository? Že se na začátku spočítá počet kandidátů a případně to bude no-op.

Proč vůbec potřebujeme vědět, že tam žádní nejsou, když chceme ukládat? Je to jen nějaká optimalizace (ať se nic nevolá, pokud nejsou kandidáti), nebo je zatím nějaká logika?

Reply:
Import musí proběhnout právě jednou. Přesunu to do repository.